api/opds.py

```diff
@@ -245,20 +267,28 @@ def annotate_work_entry(self, work, active_license_pool, edition, identifier, fe
         for tag in link_tags:
             entry.append(tag)
 
-        # Add a link for related recommendations if available.
-        if NoveListAPI.is_configured():
+        # Add a link for related books if available.
+        if self.related_books_available(active_license_pool):
             feed.add_link_to_entry(
                 entry,
                 rel='related',
                 type=OPDSFeed.ACQUISITION_FEED_TYPE,
                 title='Recommended Works',
                 href=self.url_for(
-                    'recommendations',
+                    'related_books',
```

Comment:
I think the IANA 'related' link relation would work here.

_The value "related" signifies that the IRI in the value of the href attribute identifies a resource related to the resource described by the containing element._